### ISSUE TITLE
inotifyscan hangs after some time

### ISSUE THREAD

**Issue by bjo81** [Blaok/nextcloud-inotifyscan#25]
```
Nov 06 16:37:04 mail.somehost.org nextcloud-inotifyscan[4951]: INFO - Scan for /bjo/files/Screenshots/
Nov 06 16:37:05 mail.somehost.org nextcloud-inotifyscan[4951]: INFO - Scan for /bjo/files/Screenshots/Screenshot_20231106-163657.png.ocTransferId431656977.part
Nov 07 00:35:33 mail.somehost.org nextcloud-inotifyscan[4951]: INFO - Found /data/cloud/bjo/files/ DELETE acme.log
Nov 07 00:35:33 mail.somehost.org nextcloud-inotifyscan[4951]: INFO - Found /data/cloud/bjo/files/ MOVED_TO acme.log
Nov 07 00:35:33 mail.somehost.org nextcloud-inotifyscan[4951]: INFO - Scan for /bjo/files/
```

I wondered why some files synced via Synchting were missing and had to `php /usr/share/webapps/nextcloud/occ files:scan  --all` manually.

**Comment by Blaok** [comment]
Hi @bjo81,

I actually use Syncthing with Nextcloud as well, and haven't noticed similar issues. Could you provide more details? A minimal repro would be ideal, if possible.

**Comment by bjo81** [comment]
Unfortunately I don't now what causes this issue and how to reproduce it. Like mentioned above, `Nov 07 00:35:33` was the last log entry, but files got changed afterwards bei syncthing and inotifyscan did not recognize the changes. Do I understand correctly that ` Scan for` is triggered by a change, so it should find the change afterwards?

**Comment by bjo81** [comment]
Yeah, first it also looked ok in my test. Maybe the issue was caused by sshfs/gocryptfs. I switched to juicefs now and will monitor if the issue still persists
